src/clients_api_coverage.md

```diff
@@ -0,0 +1,71 @@
+# Parsec API coverage within client libraries
+
+The current interface contracts defined for communication between Parsec and its clients is tracking
+the [PSA Cryptography API version
+1.0](https://developer.arm.com/architectures/security-architectures/platform-security-architecture/documentation).
+This page describes the current state of support within clients for the operations, attributes and
+options that are defined in that specification and for Parsec-related functionality (e.g. support
+for various providers).
+
+For an overview of the functionality supported by the service, see the [service API coverage
+page](service_api_coverage.md).
+
+**Note:** Only client libraries that are under our control are represented in the tables below. This
+does not guarantee or imply the same amount of support within other libraries. We encourage each
+library author to publish their own coverage figures in their format of choice. Links will be added
+here for any other such coverage report.
+
+## Provider support
+
+| Provider    | Rust client |
+|-------------|-------------|
+| Mbed Crypto | ✅         |
+| PKCS11      | ✅         |
+| TPM 2.0     | ✅         |
+
+## Operation support
+
+| Operation         | Rust client |
+|-------------------|-------------|
+| Import key        | ✅         |
+| Generate key      | ✅         |
+| Export public key | ✅         |
+| Destroy key       | ✅         |
+| Sign hash         | ✅         |
+| Verify hash       | ✅         |
+
+## Algorithm support
+
+### Hash algorithms
+
+| Algorithm   | Rust client |
+|-------------|-------------|
+| MD2         | ✅         |
+| MD4         | ✅         |
+| MD5         | ✅         |
+| RIPEMD160   | ✅         |
+| SHA-1       | ✅         |
+| SHA-224     | ✅         |
+| SHA-256     | ✅         |
+| SHA-384     | ✅         |
+| SHA-512     | ✅         |
+| SHA-512-224 | ✅         |
+| SHA-512-256 | ✅         |
+| SHA3-224    | ✅         |
+| SHA3-256    | ✅         |
+| SHA3-384    | ✅         |
+| SHA3-512    | ✅         |
+| Any hash    | ✅         |
```

Comment:
"Any hash" can be removed now I think as it has been removed from the operations